src/zope/publisher/browser.py

```diff
@@ -630,16 +627,8 @@ def get(self, key, default=None):
 
         return super(BrowserRequest, self).get(key, default)
 
-class ZopeFieldStorage(FieldStorage):
-
-    def make_file(self, binary=None):
-        if PYTHON2 or self._binary_file:
-            return tempfile.NamedTemporaryFile("w+b")
-        else:
-            return tempfile.NamedTemporaryFile("w+",
-                    encoding=self.encoding, newline='\n')
-
```

Comment:
Maybe add a compatibility alias `ZopeFieldStorage = FieldStorage`, in case something imports and uses it?

(Nothing in zopefoundation/ does this, but there might packages elsewhere.  z3c.soap does import ZopeFieldStorage, but from ZPublisher.HTTPRequest via a star import.)

Reply:
I've added a `from cgi import FieldStorage as ZopeFieldStorage`, ignoring `ImportError`s.  Is that the sort of thing you meant?

Reply:
Ok, I think what happened was I was under the misapprehension that the code here subclassed cgi.FieldStorage into a FieldStorage and then had a 2nd subclass as ZopeFieldStorage, for the purposes of I don't even know what.

I'm -1 for making ZopeFieldStorage available as an alias for cgi.FieldStorage if this changes behavior of ZopeFieldStorage from previous zope.publisher versions.  Let's just drop it.